html/wp-content/themes/ailurotech/index.php

```diff
@@ -48,6 +48,25 @@
 $context['button_href'] = get_field('buttonHref');
 $context['contact_content'] = get_field('content');
 $context['contact_title'] = get_field('title');
+//feature section
+
+
+$feature_icon=get_field('feature_icon');
+$feature_title=get_field('feature_title');
+
+$feature_left_title=get_field('feature_left_title');
+$feature_left_details=get_field('feature_left_details');
+$feature_right_details_title=get_field('feature_right_details_title');
+$feature_right_details_number=get_field('feature_right_details_number');
+
+$context['feature_title']=get_field('feature_title');
+$context['feature_right_title']=get_field('feature_right_title');
+$context['feature_icon'] = explode(" ", $feature_icon);
```

Comment:
Can you make these consistent? This is very confusing when sometimes sepreated by space, !, and ,